compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2736,11 +2736,15 @@ static bool enablePrimitiveArrayCopyInlineSmallSizeWithoutREPMOVS(uint8_t elemen
    if (!cg->comp()->target().cpu.supportsAVX() || !cg->comp()->target().is64Bit())
       return false;
 
+   static bool disableAVXUseInArrayCopy = feGetEnv("TR_DisableAVXUseInArrayCopy") != NULL;
    static bool disable8BitPrimitiveArrayCopyInlineSmallSizeWithoutREPMOVS  = feGetEnv("TR_Disable8BitPrimitiveArrayCopyInlineSmallSizeWithoutREPMOVS") != NULL;
    static bool disable16BitPrimitiveArrayCopyInlineSmallSizeWithoutREPMOVS = feGetEnv("TR_Disable16BitPrimitiveArrayCopyInlineSmallSizeWithoutREPMOVS") != NULL;
    static bool disable32BitPrimitiveArrayCopyInlineSmallSizeWithoutREPMOVS = feGetEnv("TR_Disable32BitPrimitiveArrayCopyInlineSmallSizeWithoutREPMOVS") != NULL;
    static bool disable64BitPrimitiveArrayCopyInlineSmallSizeWithoutREPMOVS = feGetEnv("TR_Disable64BitPrimitiveArrayCopyInlineSmallSizeWithoutREPMOVS") != NULL;
 
+   if (disableAVXUseInArrayCopy || cg->comp()->getOption(TR_DisableAVXUseInArrayCopy))
```

Comment:
Isn't this more than just disabling AVX in arraycopy ? 

I mean, an arraycopy may be short enough that it was never going to use AVX to begin with, but this option would disable the fast path in such cases too ?